packages/xdl/src/detach/AndroidIntentFilters.ts

```diff
@@ -26,7 +26,7 @@ export default function renderIntentFilters(intentFilters: any) {
 }
 
 function renderIntentFilterDatumEntries(datum: any) {
-  return _.toPairs(datum)
+  return toPairs(datum)
```

Comment:
```suggestion
  return Object.entries(datum)
```
Hard to tell from the `any` types, but prolly can use `Object.entries`.